embedded-aerospike/src/main/java/com/playtika/test/aerospike/EmbeddedAerospikeBootstrapConfiguration.java

```diff
@@ -39,17 +44,40 @@ public AerospikeWaitStrategy aerospikeStartupCheckStrategy(AerospikeProperties p
         return new AerospikeWaitStrategy(properties);
     }
 
+    @Bean
+    @Conditional(AerospikeToxiProxyProxyEnabled.class)
+    ToxiproxyContainer.ContainerProxy aerospikeContainerProxy(ToxiproxyContainer toxiproxyContainer,
```

Comment:
I like this approach with external proxy so we're not tied to a single way of communicating with the container from the outside.

Reply:
thanks. I think, that separate properties for the proxy are more transparent for the end user.